core/unit_test/TestExecSpacePartitioning.hpp

```diff
@@ -75,23 +76,61 @@ void check_distinctive(Kokkos::Experimental::SYCL exec1,
             *exec2.impl_internal_space_instance()->m_queue);
 }
 #endif
+#ifdef KOKKOS_ENABLE_OPENMP
+void check_distinctive(Kokkos::OpenMP exec1, Kokkos::OpenMP exec2) {
```

Comment:
Distinct or distintive?
Considering that you can directly compare them now, I would say drop the function.

Reply:
(OK now I see you didn't come up with the name.  Regardless I think we should get rid of that function.

Reply:
I quickly tried to remove this function and it doesn't work out of the box. Let's do this in a different PR.